### DISCUSSION TITLE
데이터 조직의 다른 부서 대상 교육, 어떤 것들을 진행하시나요?

### DISCUSSION THREAD

**Discussion by BokyungChoi** [playinpap/awesome-data-and-analytics-governance#16] [Q&A]
다른 부서에 또는 전사 대상으로 교육을 진행 하시는 데이터 팀이 계신가요? 
어떤 것부터 시작하셨고, 어떤 절차로 진행하셨나요? 궁금합니다. 

> SQL? 대시보드 활용? 실험? 지표? 데이터 파이프라인? 사내 툴이 있다면 툴에 대한 교육?

아마 많은 분들이 SQL부터 차근차근 시작하셨을 것 같은데, 그 이후에는 어떤 교육들을 하셨는지도 궁금하네요.
주로 협업하는 조직들에서 니즈가 있는 것부터 교육을 진행하실 것 같아서, 정답이나 Best Practice가 없을 수 있겠지만 다른 분들의 경험을 참고하고 싶습니다.

가볍게 각자 경험을 알려주고 가시면 좋을 것 같아요.

**Comment by jsaang** [comment]
교육이라고 표현하기엔 빈약하지만 아래와 같은 주제들로 세션을 진행한 경험이 있습니다.

- **현재 우리 회사가 어떠한 데이터를 가지고 있는가?** : SQL과 같은 데이터 사용 툴을 알아도 어떤 데이터가 어디에 있는지 모르면 아무 소용이 없기에, 간단하게라도 회사가 수집하고 있는 데이터를 구성원들에게 소개하는 것이 의미있다고 생각했습니다.
- **데이터 분석가는 어떤 프로세스를 거쳐 분석을 수행하고 무엇을 고민하는가?** : 직군간 원활한 협업은 서로의 업에 대한 이해로부터 나온다고 생각합니다. 때문에 분석가가 어떤 고민과 업무 경험을 하는지 설명하고, 데이터 분석가와 협업할 때 고려해주시면 좋은 부분에 대해 설명드렸었습니다.
- **현재 서비스에 적용되고 있는 몇몇 ML 모델들은 어떻게 동작하는가?**
